pkg/capabilities/consensus/ocr3/reporting_plugin.go

```diff
@@ -181,6 +186,13 @@ func (r *reportingPlugin) Outcome(outctx ocr3types.OutcomeContext, query types.Q
 			countedWorkflowIDs[id] = true
 		}
 
+		// Count the frequency of timestamps in the observations.
```

Comment:
You will most likely get N different answers, each with frequency=1. Let's do a median instead.